package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "xregexp": "^5.1.1"
+  }
+}
```

Comment:
Tohle teda odebrat a dát přímo k webu (a osobně se příkláním k tomu že to budeme odebírat, nevím úplně o důvodu proč tu mít regex pomocnou knihovnu)

Reply:
Odstraněno